kong/conf_loader.lua

```diff
@@ -182,6 +182,12 @@ local function check_and_infer(conf)
      errors[#errors+1] = "must specify 'cassandra_local_datacenter' when "..
                         "DCAwareRoundRobin policy is in use"
   end
+  for _, contact_point in pairs(conf.cassandra_contact_points) do
+    if contact_point:find(':') then
+      errors[#errors+1] = "can't specify port in cassandra_contact_point, "..
+                         "use cassandra_port for that"
+    end
+  end
```

Comment:
also some tests are missing in the validation section here: https://github.com/Mashape/kong/blob/f9716acfe4f1c1417c747bad02aa01909ebb3208/spec/01-unit/02-conf_loader_spec.lua#L179